pkg/pbutil/output.go

```diff
@@ -64,3 +65,28 @@ func FTextPB(w io.Writer, m protoreflect.ProtoMessage) error {
 	_, err = w.Write(mt)
 	return err
 }
+
+func PB(m protoreflect.ProtoMessage, filename string, fs afero.Fs) error {
```

Comment:
Nit: Can you add a comment explaining what these funcs do? especially since the name isnot self-descriptive. Also, this might be the Java in me, but why not name it something like MarshalBinary, MarshalText, etc.?

Reply:
`func FPB` transforms protocol buffer message to a binary message and writes to a IO writer.
`func PB` transforms protocol buffer message to a binary message and output to a file(IO writer).
And yes, the name is not self-descriptive as I followed existing naming convention like `FTextPB`, `TextPB`, `FJSONPB` and `JSONPB`. 
And I have renamed them and added comments.